sdk/storage/azblob/CHANGELOG.md

```diff
@@ -0,0 +1 @@
+# Change Log
```

Comment:
Add an entry that says something similar to "This is the first public preview", refer to sdk/data/aztables for an example

Reply:
The CHANGELOG.md files need to follow the default format as its read by automation.

```
# Release History

## v0.1.0 (Unreleased)

### Features Added

### Breaking Changes

### Bugs Fixed

### Other Changes
```